cogs/csgo.py

```diff
@@ -158,9 +158,9 @@ async def pug(self, ctx):
         team2_steamIDs = []
 
         team1_channel = await ctx.author.voice.channel.category.create_voice_channel(
-            name=f'{team1_captain.display_name}\'s Team', user_limit=7)
+            name=f'team_{team1_captain.display_name}', user_limit=7)
         team2_channel = await ctx.author.voice.channel.category.create_voice_channel(
-            name=f'{team2_captain.display_name}\'s Team', user_limit=7)
+            name=f'team_{team1_captain.display_name}', user_limit=7)
```

Comment:
Should be `team_{team2_captain.display_name}`

Reply:
Fixed it commit